build.assets/Dockerfile-grpcbox

```diff
@@ -10,16 +10,16 @@ RUN apt-get update && \
 
 # protoc-gen-gogofast
 # Keep in sync with api/proto/buf.yaml (and buf.lock)
-ARG GOGO_PROTO_TAG # eg, "v1.3.2"
+ARG GOGO_PROTO_TAG
```

Comment:
Can we keep the example versions?
If they are not supported at the end of the line, let's move them one line up
```suggestion
# eg, "v1.3.2"
ARG GOGO_PROTO_TAG
```

Reply:
Added